### ISSUE TITLE
CVE-2022-40320

### ISSUE THREAD

**Issue by RobertBerger** [libconfuse/libconfuse#177]
The last tagged release (3.3) contains CVE-2022-40320[1]

This is fixed here [2].

Maybe a new release (3.4?) would be useful so we don't need to carry a private patch against 3.3 in various distros.

[1] https://security-tracker.debian.org/tracker/CVE-2022-40320
[2] https://github.com/libconfuse/libconfuse/commit/d73777c2c3566fb2647727bb56d9a2295b81669b

**Comment by RobertBerger** [comment]
My proposed patch against 3.3 would be something like this

[0001-fix-for-CVE-2022-40320.patch.tar.gz](https://github.com/user-attachments/files/17692684/0001-fix-for-CVE-2022-40320.patch.tar.gz)

**Comment by troglobit** [comment]
Yeah sorry about that, this project has been a very low priority for me for a few years.  Life and all that ... that will change now, with a 3.4 release just around the corner. I'm still testing against a few active projects, expect the GA to be out in a few days.

Closing now, so I don't forget after the release.
